editors/tree-sitter-slint/grammar.js

```diff
@@ -475,25 +477,63 @@ module.exports = grammar({
             seq(
                 optional(field("purity", "pure")),
                 "callback",
-                field("name", $.function_identifier),
+                field("name", $.var_identifier),
                 "<=>",
                 field("alias", $.var_identifier),
                 ";",
             ),
 
         callback_event: ($) =>
             seq(
-                field("name", $.function_identifier),
+                field("name", $.var_identifier),
                 optional($.call_signature),
                 "=>",
                 field("action", $.block),
             ),
 
         function_call: ($) =>
+            seq(field("name", $.var_identifier), optional($.call_signature)),
```

Comment:
With this in place - which makes sense - is there any use of $.function_identifier left?